dev/plugins/hu.elte.txtuml.export.javascript/.gitignore

```diff
@@ -0,0 +1,3 @@
+bin/
+target/
+src/hu/elte/txtuml/export/javascript/resources/res/input.js
```

Comment:
That was an exported diagram (normaly this file is generated in the target project), it was copied there to help faster development of the visualizer. It was added to .gitignore to prevent an accidental commit. The resource is no longer there but I forgot about the .gitignore file, I will remove it.